kong/plugins/acme/handler.lua

```diff
@@ -221,6 +221,12 @@ function ACMEHandler:access(conf)
     end
 
     if captures then
+      -- creating account through proxy side with "kong" storage in Hybrid mode is not supported
+      -- if this is just a sanity test, we always return 200 status
+      if captures[1] == "x" and kong.configuration.role == "data_plane" and conf.storage == "kong" then
+        return kong.response.exit(200, { message = "ok" })
```

Comment:
If it's impossible to go through the process, should we instead return 422 or something?

Reply:
@fffonion I believe we got it wrong yesterday. The correct fix should be to always return 404 instead of 200.